applications/luci-app-mwan3/luasrc/controller/mwan3.lua

```diff
@@ -301,7 +301,7 @@ function troubleshootingData()
 	mArray.iprule = { ipRuleShow }
 
 	-- ip route list table 1-250
-	local routeList, routeString = ut.trim(sys.exec(ip .. "rule | sed 's/://g' | awk '$1>=2001 && $1<=2250' | awk '{print $NF}'")), ""
+	local routeList, routeString = ut.trim(sys.exec(ip .. "rule | sed 's/://g' 2>/dev/null | awk '$1>=2001 && $1<=2250' | awk '{print $NF}'")), ""
```

Comment:
to prevent sed write errors from being logged. this fixes https://github.com/openwrt/packages/issues/4246